website/settings/defaults.py

```diff
@@ -52,12 +52,15 @@ def parent_dir(path):
 DEV_MODE = False
 DEBUG_MODE = False
 
-# Should be True in local development and False on staging, staging2, production
+# True if in local development and False if on staging, staging2, production
```

Comment:
Note to self: the distinguishing feature of local mode is that someone is running via the built-in testing server of the framework (`app.run`, `manage.py runserver`, etc) rather than an external server via wsgi.